ouroboros-consensus/test-util/Test/Util/TestBlock.hs

```diff
@@ -298,6 +299,12 @@ instance UpdateLedger TestBlock where
 
   ledgerTipPoint = lastAppliedPoint
 
+instance HasAnnTip TestBlock where
+  -- Use defaults
+
+instance ValidateEnvelope TestBlock where
+  firstBlockNo _ = Block.BlockNo 1
```

Comment:
Comment why this is necessary (block number is derived from hash and starts at 1)